diff -Naur jflex-1.4.3.orig/src/JFlex/tests/AllTests.java jflex-1.4.3/src/JFlex/tests/AllTests.java
--- jflex-1.4.3.orig/src/JFlex/tests/AllTests.java	2009-01-31 09:52:34.000000000 +0100
+++ jflex-1.4.3/src/JFlex/tests/AllTests.java	2012-03-12 13:47:35.835005672 +0100
@@ -32,7 +32,7 @@
 public class AllTests {
 
   public static void main(String[] args) {
-    junit.textui.TestRunner.run(AllTests.class);
+//    junit.textui.TestRunner.run(AllTests.class);
   }
 
   public static Test suite() {
